nf/CELLRANGER/modules/local/tar/main.nf

```diff
@@ -0,0 +1,15 @@
+process UNTAR_REF {
+    label 'C2'
+    container 'ubuntu:latest'
+
+    input:
+    path(tar_file)
+
+    output:
+    path("${tar_file.name.substring(0, tar_file.name.length() - ".tar.gz".length())}")
```

Comment:
If the tar doesn't have any preceding `.` you can use simpleName. Otherwise you could do `${tar_file.replaceFirst(/\.tar\.gz$/, "")}`  
```suggestion
    path("${tar_file.simpleName}")
```